pkg/cmd/admin/registry/registry.go

```diff
@@ -426,6 +428,18 @@ func (opts *RegistryOptions) RunCmdRegistry() error {
 				},
 			},
 		})
+	} else if opts.Config.Deployment {
```

Comment:
We may not use by default hooks or triggers in these deployments but what if someone wants to customize them to do so? For example, set a post hook to send e-mails after the registry is deployed? Or pick up the images to deploy from a private registry (ICT)?